js-miniapp-sdk/src/miniapp.ts

```diff
@@ -14,13 +15,36 @@ interface MiniApp {
  * A contract declaring the interaction mechanism between mini-apps and native host app to display ads.
  */
 interface Ad {
-  /** @returns The Promise of interstitial ad response result from injected side. */
-  showInterstitialAd(): Promise<InterstitialAdResponse>;
+  /**
+   * Loads the specified Interstittial Ad Unit ID.
+   * Can be called multiple times to pre-load multiple ads.
+   * Promise is resolved when successfully loaded.
+   * Promise is rejected if failed to load.
```

Comment:
`@returns` can also be added into the doc.

Reply:
Added @returns decorator